step1/react-todo/src/App.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useRef, useState } from "react";
+import Header from "./components/Header";
+import TodoInput from "./components/TodoInput";
+import TodoList from "./components/TodoList";
+import { ApolloProvider, gql, useQuery, useReactiveVar } from "@apollo/client";
+
+import TodoItem from "./components/TodoItem";
```

Comment:
```suggestion
import { ApolloProvider, gql, useQuery, useReactiveVar } from "@apollo/client";
import { Header, TodoInput, TodoList, TodoItem } from "./components";
```
이렇게 표현할 수 있도록 components에 `index.ts` 를 만들어서 관리해보면 어떨까요?